api/repository.go

```diff
@@ -85,6 +85,7 @@ type RepositoryFind struct {
 	// Related fields
 	VCSID     *int
 	ProjectID *int
+	WebURL    *string
```

Comment:
this should be in the domain specific fields section

Reply:
Done.